src/input_cp2k_dft.F

```diff
@@ -6188,7 +6195,7 @@ SUBROUTINE create_print_embed_pot_cube(section)
                           description="The stride (X,Y,Z) used to write the cube file "// &
                           "(larger values result in smaller cube files). You can provide 3 numbers (for X,Y,Z) or"// &
                           " 1 number valid for all components.", &
-                          usage="STRIDE 1 1 1", n_var=-1, default_i_vals=(/1, 1, 1/), type_of_var=integer_t)
+                          usage="STRIDE 2 2 2", n_var=-1, default_i_vals=(/2, 2, 2/), type_of_var=integer_t)
```

Comment:
This is not restricted to Xas, correct? Is changing the defaults here intentional? If yes, please factor out to separate commit.

Reply:
Correct, I am not sure how it got there. I'll fix that